docs/release-notes/v11.7.0.md

```diff
@@ -41,6 +41,36 @@ This version addressed a few bugs, here is a comprehensive list:
 
 ### Fast Data Services
 
+#### Single View Trigger Generator
+
+The new version `v3.1.2` of the `single-view-trigger-generator` is available!
+
+##### Improvements
+
+###### Support for `pr-update` v2
+
+This version introduces the support for reading _projection record update events_ that are using the schema version 2, which
+can be employed either as drop in or alongside the current _projection record update events_ in version v1.
+
+##### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* support for custom CA certificate in the Kafka Producer client
+* projection record update events with operation type `DELETE` and the `after` property set to `null` are now processed correctly, avoiding service restart.
+
+For more information please check out the service [changelog](/runtime_suite/single-view-trigger-generator/changelog.md)
+
+#### Ingestion Storer
+
+The new version `v1.5.1` of the `ingestion-storer` is available!
+
+##### Bug Fix
+
+This version addressed a few bugs, here is a comprehensive list:
+
+* ingestion messages with an empty or null key are now processed correctly as all the other messages. The service now does not restart in case it consume them. 
```

Comment:
```suggestion
* ingestion messages with an empty or null key are now processed correctly as all the other messages. The service now does not restart in case it consumes them. 
```